beam/tests/setup.py

```diff
@@ -1,577 +1,588 @@
+		work.production_capacity = ws[1]
+		work.save()
+
+
+def create_operations():
+	for op in operations:
+		if frappe.db.exists("Operation", op[0]):
+			continue
+		oper = frappe.new_doc("Operation")
+		oper.name = op[0]
+		oper.workstation = op[1]
+		oper.batch_size = op[2]
+		oper.description = op[3]
+		oper.save()
+
+
+def create_item_groups(settings):
+	for ig_name in (
+		"Baked Goods",
+		"Bakery Supplies",
+		"Ingredients",
+		"Bakery Equipment",
+		"Sub Assemblies",
+	):
+		if frappe.db.exists("Item Group", ig_name):
+			continue
+		ig = frappe.new_doc("Item Group")
+		ig.item_group_name = ig_name
+		ig.parent_item_group = "All Item Groups"
+		ig.save()
+
+
+def create_items(settings):
+	if not frappe.db.exists("Price List", "Bakery Buying"):
+		pl = frappe.new_doc("Price List")
+		pl.price_list_name = "Bakery Buying"
+		pl.buying = True
+		pl.append("countries", {"country": "United States"})
+		pl.save()
+
+	if not frappe.db.exists("Price List", "Bakery Wholesale"):
+		pl = frappe.new_doc("Price List")
+		pl.price_list_name = "Bakery Wholesale"
+		pl.selling = True
+		pl.append("countries", {"country": "United States"})
+		pl.save()
+
+	if not frappe.db.exists("Pricing Rule", "Bakery Retail"):
+		pr = frappe.new_doc("Pricing Rule")
+		pr.title = "Bakery Retail"
+		pr.selling = True
+		pr.apply_on = "Item Group"
+		pr.company = settings.company
+		pr.margin_type = "Percentage"
+		pr.margin_rate_or_amount = 2.00
+		pr.valid_from = settings.day
+		pr.for_price_list = "Bakery Wholesale"
+		pr.append("item_groups", {"item_group": "Baked Goods"})
+		pr.save()
+
+	for item in items:
+		if frappe.db.exists("Item", item.get("item_code")):
+			continue
+		i = frappe.new_doc("Item")
+		i.item_code = i.item_name = item.get("item_code")
+		i.item_group = item.get("item_group")
+		i.stock_uom = item.get("uom")
+		i.description = item.get("description")
+		i.maintain_stock = True
+		i.enable_handling_unit = i.item_code not in ("Water", "Ice Water")
+		i.include_item_in_manufacturing = True
+		i.default_warehouse = settings.get("warehouse")
+		i.default_material_request_type = (
+			"Purchase" if item.get("item_group") in ("Bakery Supplies", "Ingredients") else "Manufacture"
+		)
+		i.valuation_method = "FIFO"
+		i.is_purchase_item = item.get("item_group") in ("Bakery Supplies", "Ingredients")
+		i.is_sales_item = item.get("item_group") == "Baked Goods"
+		i.append(
+			"item_defaults",
+			{"company": settings.company, "default_warehouse": item.get("default_warehouse")},
+		)
+		if i.is_purchase_item and item.get("supplier"):
+			i.append("supplier_items", {"supplier": item.get("supplier")})
+		if i.item_code == "Parchment Paper":
+			i.append("uoms", {"uom": "Box", "conversion_factor": 100})
+			i.purchase_uom = "Box"
+		if i.item_code in ("Water", "Ice Water"):
+			i.append("uoms", {"uom": "Gallon Liquid (US)", "conversion_factor": 15.142})
+			i.purchase_uom = "Gallon Liquid (US)"
+			i.valuation_rate = 0.01 if i.item_code == "Water" else 0.02
```

Comment:
@agritheory there was some conflict in the HU tests, but I wasn't sure which changes to include, so I've added both sets of changes. Can you have a look please?